src/app/static/src/app/components/adr/ADRIntegration.vue

```diff
@@ -1,7 +1,15 @@
 <template>
     <div v-if="loggedIn" class="mb-5">
         <adr-key></adr-key>
-        <select-dataset v-if="key"></select-dataset>
+        <div v-if="key">
+            <select-dataset></select-dataset>
+            <div class="pt-3" id="adr-capacity" v-if="selectedDataset">
+                <span class="font-weight-bold align-self-stretch" v-translate="'adrAccessLevel'"></span>
+                <span v-tooltip="handleUploadPermission(hasUploadPermission, true)">
+                    <a href="#">{{ handleUploadPermission(hasUploadPermission, false) }}</a>
```

Comment:
It's a bit confusing for this to be a link, could it just be a red span?

Reply:
I agree, I found myself clicking on it to see what would happen.